modules/gapi/perf/cpu/gapi_imgproc_perf_tests_cpu.cpp

```diff
@@ -126,21 +126,22 @@ INSTANTIATE_TEST_CASE_P(SobelPerfTestCPU32F, SobelPerfTest,
 
 INSTANTIATE_TEST_CASE_P(LaplacianPerfTestCPU, LaplacianPerfTest,
                         Combine(Values(AbsExact().to_compare_f()),
-                                Values(CV_8UC1, CV_8UC3, CV_16SC1),
-                                Values(3, 5),
+                                Values(CV_8UC1, CV_8UC3, CV_16UC1, CV_16SC1),
+                                Values(1, 3, 5),
                                 Values(szVGA, sz720p, sz1080p),
-                                Values(-1, CV_16S, CV_32F),
+                                Values(-1),
+                                Values(1.0),
```

Comment:
So, as I get it right, you added one more parameter but it's actually not changing during tests (and in GPU tests it's the same). Maybe you could just hardcode it in the test body?

Reply:
Decided to leave it as is